corehq/apps/reports/filters/users.py

```diff
@@ -138,11 +140,13 @@ def user_type_tuple(self, t):
 
     def location_tuple(self, location):
         location_id = location.location_id
-        text = location.get_path_display()
+        use_location_display_name = USE_LOCATION_DISPLAY_NAME.enabled(get_request_domain())
+        text = location.display_name if use_location_display_name else location.get_path_display()
+        tooltip = location.get_path_display() if use_location_display_name else location.display_name
         if self.namespace_locations:
             location_id = f'l__{location_id}'
             text = f'{text} [location]'
-        return (location_id, text)
+        return (location_id, text, None, tooltip)
```

Comment:
ya, it is used in a lot of places (although I think `_CallCenterOwnerOptionsUtils` uses an unrelated `location_tuple`) While I agree that avoiding changes to existing code can be a safer approach, I think this code logically belongs here (like if this was from scratch, i wouldn't have had two separate location tuple methods). I would want to be cautious about making it a general practice to avoid modifying existing code out of concern of breaking things. 

But it does mean I need to be more cautious with checking for ripple effects, so this time around i searched for usages of `location_tuple` to check it won't be affected by the additional return elements.

Reply:
Although, thinking about this in the context of our other location changes, maybe it would be prudent to put this behind the feature preview flag. So it'll continue to return `return (location_id, text` if the flag is off. That would mitigate risks with this

Reply:
Thank you for double checking the references, that definitely gives me more confidence too that i didn't miss something.